### DISCUSSION TITLE
Elsa 3 - How to setup project using net 8 alsa studio blazor server app

### DISCUSSION THREAD

**Discussion by nurhidayat-agung** [elsa-workflows/elsa-core#5609] [Q&A]
hello elsa ,
How to setup project using net 8 alsa studio blazor server app due in the documentation using net 7 and whne implemnt in net 8 template i got error in Severity	Code	Description	Project	File	Line	Suppression State	Details
Error (active)	RZ9978	The directives @addTagHelper, @removeTagHelper and @tagHelperPrefix are not valid in a component document. Use '@using <namespace>' directive instead. in	_Host.razor file

**Comment by glime-ai** [comment]
To set up an Elsa Studio Blazor Server app using .NET 8, you will need to follow the documentation for .NET 7 and make adjustments as necessary for .NET 8. Since .NET 8 is not officially released at the time of my knowledge cutoff in 2023, there may not be specific documentation for it. However, you can try the following steps to resolve the issue you're encountering:

1. Ensure that you have the correct .NET 8 SDK installed on your machine.
2. Create a new Blazor Server application targeting .NET 8.
3. Add the necessary Elsa Studio packages to your project.
4. Modify the `Program.cs` file as per the documentation for .NET 7, but make sure to adjust any startup configuration that may have changed in .NET 8.
5. Remove any unnecessary files as instructed in the documentation.
6. Update the `appsettings.json` file with the required settings.
7. Update the `_Host.cshtml` file. If you encounter the error `RZ9978`, it suggests that you are using directives that are not valid in a component document. Replace `@addTagHelper`, `@removeTagHelper`, and `@tagHelperPrefix` directives with the appropriate `@using <namespace>` directive.

Since the error you're seeing is related to tag helpers in a Blazor component, you should ensure that you're using the correct syntax for .NET 8. If the syntax has changed from .NET 7 to .NET 8, you will need to update your code accordingly.

If you continue to encounter issues, you may need to refer to the official .NET documentation for any changes in Blazor Server apps between .NET 7 and .NET 8, or seek assistance from the Elsa community or support channels.

Please note that since .NET 8 is not officially released, you may encounter issues that are due to changes in the framework that are not yet documented or supported by third-party libraries like Elsa. It's also possible that Elsa Studio may not yet be compatible with .NET 8, in which case you might need to wait for an official update from the Elsa team.

If you provide the exact error messages or additional context, I may be able to offer more targeted assistance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
